docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations.rst

```diff
@@ -435,6 +435,8 @@ Beginning in version 0.13, we have introduced a new API focused on enabling Modu
 
         This guide will walk you through the process of creating your own Modular Expectations in 6 simple steps!
 
+        See also this `complete example <https://github.com/superconductive/ge_tutorials/tree/main/getting_started_tutorial_final_v3_api/great_expectations/plugins/column_custom_max_expectation.py>`_.
```

Comment:
Before https://github.com/superconductive/ge_tutorials/pull/21 is merged, this can be viewed at https://github.com/jdimatteo/ge_tutorials/blob/example_custom_expectation/getting_started_tutorial_final_v3_api/great_expectations/plugins/column_custom_max_expectation.py